edge-modules/api-proxy-module/src/monitors/utils.rs

```diff
@@ -0,0 +1,26 @@
+use std::{fs::{self, File}, io::prelude::*, sync::Arc};
+use anyhow::{Context, Result};
+use tokio::sync::Notify;
+
+pub fn write_binary_to_file(file: &[u8], path: &str) -> Result<()> {
```

Comment:
```suggestion
pub fn write_binary_to_file<P:AsRef<Path>>(path: P, content: &[u8]) -> Result<()> {
```

Reply:
I need to discuss that with you, I don't understand.